### ISSUE TITLE
Keywords and categories

### ISSUE THREAD

**Issue by michael** [substance/texture#1270]
# Description

Please provide a brief description of this feature.

# User stories

Please add your user stories.

# Use cases

- Article keywords (needs spec)
- Article subjects (needs spec)
- Figure metadata field editing
- Contribution roles

# But what if . . . ?

Please provide details of situations that may occur and/or overlap with other features and are not covered in the user stories.

# Consideration

Please add any additional requirements you may have.

# XML requirements

Figure Metadata keywords:

```xml
<fig>
<kwd-group kwd-group-type="exp-system">
  <label>Exp. System</label>
  <kwd>immunostaining</kwd>
  <kwd>confocal microscopy</kwd>
</kwd-group>
<kwd-group kwd-group-type="measured-variables">
  <label>Measured-Variables</label>
  <kwd>immunostaining</kwd>
  <kwd>confocal microscopy</kwd>
</kwd-group>
</fig>
```

Article keywords:

```xml
<kwd-group kwd-group-type="keyword">
  <kwd>optogenetics</kwd>
  <kwd>two-photon</kwd>
  <kwd>calcium imaging</kwd>
  <kwd>Mouse</kwd>
</kwd-group>
<kwd-group kwd-group-type="research-organism">
  <kwd>mouse</kwd>
</kwd-group>
```

# Mock ups

This is not required, but if you have mock ups of what you would like to see please provide them here. 

# Proposal

This will be added by the Texture team after the feature request is discussed and agreed.

**Comment by michael** [comment]
@Melissa37 please add your user stories here.

**Comment by Melissa37** [comment]
@michael I have pulled this out of the ticket but it is kept here for now. They cannot be in the ticket without user stories attached to them.

## Figure Metadata keywords:

```xml
<fig>
<kwd-group kwd-group-type="exp-system">
  <label>Exp. System</label>
  <kwd>immunostaining</kwd>
  <kwd>confocal microscopy</kwd>
</kwd-group>
<kwd-group kwd-group-type="measured-variables">
  <label>Measured-Variables</label>
  <kwd>immunostaining</kwd>
  <kwd>confocal microscopy</kwd>
</kwd-group>
</fig>
```

**Comment by oliver----** [comment]
@Melissa37 I sense some source of confusion re `<fig>` level keywords. IMO the user-stories are essentially the same as for '<article-meta>' level keywords. Also there we might consider more scopes for keywords, such as `table-wrap` or `supplementary-material`. Wouldn't it make sense to cover all of such cases with one set of user-stories, and add refinements where necessary?

**Comment by Melissa37** [comment]
Yes I agree. Shall we pull in the figure metadata ones here and close the ticket 
https://github.com/substance/texture/issues/1249

The tickets we're generating have good user stories that have been verified by Nick, so we'd like to keep that process going.
Also, we want to provide the Consortium tickets as we produce them and I am happy to pull the ScourceData use cases and examples into them, with good user stories added, but we can't have our user stories generated by anyone else and before we get to them :-)

**Comment by JGilbert-eLife** [comment]
@Melissa37, @michael, @source-data - I have added user stories related to figure-specific keywords here. I am assuming these will be at the panel level, but will update if that's not right.
